product_last_purchase_date/models/product_product.py

```diff
@@ -29,7 +29,7 @@ def _compute_last_purchase_date(self):
                 self._get_move_domain(product), order="id desc", limit=1
             )
             if move:
-                last_purchase_date = fields.Date.context_today(self, move.date)
+                last_purchase_date = move.accounting_date
```

Comment:
If the accounting_date is not set, won't anything be updated?

Reply:
With the design of computing accounting_date, it will always have value.
https://github.com/qrtl/axls-oca/blob/4085c38bc89d12ee9afa1194283d16191b4d00c8/stock_picking_accounting_date/models/stock_move.py#L18-L21